lib/pos_server_regtest.go

```diff
@@ -90,17 +90,3 @@ func (srv *Server) submitRegtestValidatorRegistrationTxns(block *MsgDeSoBlock) {
 		}
 	}
 }
-
-func (srv *Server) startRegtestFastHotStuffConsensus(block *MsgDeSoBlock) {
```

Comment:
This hook was a special case we previously used to start the consensus on regtest. We don't need it anymore now that we have the generalized transition detector introduced by this PR